### ISSUE TITLE
Web site/documentation Trivia

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#408]
```
The railroad diagram for Constraint shows the line leaving for primary key 
before <constraint name definition>, but it should actually leave after it.


```

Original issue reported on code.google.com by `Alan.L.C...@gmail.com` on 18 Jul 2012 at 11:03

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thanks! This will be fixed in the next release.

Regards,
Thomas
```

Original comment by `thomas.t...@gmail.com` on 22 Jul 2012 at 6:22
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in version 1.3.169. Please note you will need to refresh 
the web site (possibly clear the browser cache) as the railroad images have 
changed (to support the Apple retina display).
```

Original comment by `thomas.t...@gmail.com` on 9 Sep 2012 at 12:24
- Changed state: **Fixed**
